src/modules/dashboard/components/ManageReputationDialog/ManageReputationDialog.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import { ColonyRole } from '@colony/colony-js';
+
+import { DialogProps, ActionDialogProps } from '~core/Dialog';
+import IndexModal from '~core/IndexModal';
+
+import { WizardDialogType, useTransformer } from '~utils/hooks';
+import { useLoggedInUser, Colony } from '~data/index';
+
+import { getAllUserRoles } from '../../../transformers';
+import { userHasRole } from '../../../users/checks';
+
+const MSG = defineMessages({
+  dialogHeader: {
+    id: 'dashboard.ManageReputationDialog.dialogHeader',
+    defaultMessage: 'Manage Reputation',
+  },
+  awardReputationTitle: {
+    id: 'dashboard.ManageReputationDialog.awardReputationTitle',
+    defaultMessage: 'Award reputation',
+  },
+  awardReputationDescription: {
+    id: 'dashboard.ManageReputationDialog.awardReputationDescription',
+    defaultMessage: 'Award reputation without making payments.',
+  },
+  permissionText: {
+    id: 'dashboard.ManageReputationDialog.permissionsText',
+    defaultMessage: `You must have the arbitration permission in the
+      relevant teams, in order to take this action`,
+  },
+  smiteReputationTitle: {
+    id: 'dashboard.ManageReputationDialog.smiteReputationTitle',
+    defaultMessage: 'Smite',
+  },
+  smiteReputationDescription: {
+    id: 'dashboard.ManageReputationDialog.smiteReputationDescription',
+    defaultMessage:
+      'Punish undesirable behaviour by deducting Reputation points.',
+  },
+});
+
+interface CustomWizardDialogProps extends ActionDialogProps {
+  nextStep: string;
+  prevStep: string;
+  colony: Colony;
+}
+
+type Props = DialogProps & WizardDialogType<object> & CustomWizardDialogProps;
+
+const displayName = 'dashboard.ManageReputationDialog';
```

Comment:
This is not something we should do right now, but at some point, I suggest we move all dialog components under something like: `~dashboard/Dialogs/...` and then we can remove the `Dialog` part from the component's name.

So this one would become: `/src/modules/dashboard/components/Dialogs/ManageReputation`

Thoughts @ArmandoGraterol @alicjakujawa @chinins ?

Reply:
That sounds like a good idea to me!

Reply:
Totally agree! I can create an issue for that in the maintenance project if that's ok with everybody?